### PR TITLE
doc: Add Self Observability service discovery (k8s)

### LINKED ISSUE #7259: Add doc about OTEL + OAP Self-observability
image

According to my recent reading, we missed this part in the self-observability doc. @wankai123 Could you fill this in?

### PR DESCRIPTION
**Pull request by wankai123**
- [X] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #7259.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

File: docs/en/setup/backend/backend-telemetry.md
Comment (wu-sheng) [inline docs/en/setup/backend/backend-telemetry.md:22]:
```suggestion
### Static IP or hostname
```

I think Dedicated IP would be assigned by k8s, but the root cause is they are not static, IP would change in every pod reboot or redeployment.

File: docs/en/setup/backend/backend-telemetry.md
Comment (kezhenxu94) [inline docs/en/setup/backend/backend-telemetry.md:110]: It's hard to know where to set these at the first glance, let's provide a demo command somewhere like this


```
            helm -n istio-system install skywalking skywalking \
                 --set oap.storageType='elasticsearch' \
                 --set oap.replicas=2 \
                 --set oap.ports.prometheus-port=1234 \ # <<< Expose self observability metrics port
                 --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS='k8s-mesh\,persistence' \
                 --set oap.env.SW_ENVOY_METRIC_ALS_TCP_ANALYSIS='k8s-mesh\,persistence' \
                 --set oap.env.SW_OTEL_RECEIVER=default \ # <<< Enable Otel receiver
                 --set oap.env.SW_OTEL_RECEIVER_ENABLED_OC_RULES='oap\,vm\,istio-controlplane\,k8s-cluster\,k8s-node\,k8s-service' \ # <<< Add oap analyzer for Otel metrics
                 --set oap.env.SW_TELEMETRY=prometheus \
                 --set oap.env.SW_LOG_LAL_FILES=envoy-als \
                 --set oap.envoy.als.enabled=true \
                 --set ui.image.repository=$HUB/skywalking-ui \
                 --set ui.image.tag=$TAG \
                 --set oap.image.repository=$HUB/skywalking-oap \
                 --set oap.image.tag=$TAG \
                 --set oap.env.K8S_SERVICE_NAME_RULE='${service.metadata.namespace}::${service.metadata.name}' \
                 --set elasticsearch.enabled=false \
                 --set elasticsearch.config.host=elasticsearch.istio-system.svc.cluster.local
```

**Review by kezhenxu94 — COMMENTED** [review]
And for docs updates, I think it's better to loop @wingwong-knh in to review when the docs are updated in PR rather than review the whole docs after they're merged

**Comment by wu-sheng** [comment]
> And for docs updates, I think it's better to loop @wingwong-knh in to review when the docs are updated in PR rather than review the whole docs after they're merged

I am not sure whether @wingwong-knh is available. Wing, if you are, please take a look.

File: docs/en/setup/backend/backend-telemetry.md
Comment (wankai123) [reply docs/en/setup/backend/backend-telemetry.md:110]: should this command share with the other docs? If so, I'll put it into an independent file. @kezhenxu94

**Review by wu-sheng — APPROVED** [review]
Thanks for fixing this

**Review by kezhenxu94 — APPROVED** [review]

**Comment by wingwong-knh** [comment]
> > And for docs updates, I think it's better to loop @wingwong-knh in to review when the docs are updated in PR rather than review the whole docs after they're merged
> 
> I am not sure whether @wingwong-knh is available. Wing, if you are, please take a look.

Indeed, for now I'm not sure if I can keep up with the daily updates! In the future, once all docs have been gone through once, I can start reviewing the updates in PR. For now, I'm only halfway through the backend doc (will up my pace!) For this particular doc, I will need to review from scratch.

**Comment by wu-sheng** [comment]
> > > And for docs updates, I think it's better to loop @wingwong-knh in to review when the docs are updated in PR rather than review the whole docs after they're merged
> > 
> > 
> > I am not sure whether @wingwong-knh is available. Wing, if you are, please take a look.
> 
> Indeed, for now I'm not sure if I can keep up with the daily updates! In the future, once all docs have been gone through once, I can start reviewing the updates in PR. For now, I'm only halfway through the backend doc (will up my pace!) For this particular doc, I will need to review from scratch.

Take your time. I know you are busy. We are going to continue, you could be back and polish the doc.

**Pull request merged**
